src/main/java/taxi/controller/LogoutController.java

```diff
@@ -0,0 +1,17 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+
```

Comment:
For the future, remove the first empty line in all classes/interfaces/methods where it is present

Reply:
Thank you! I'll remember it.